contracts/oracles/default/GammaPoolPriceOracle.sol

```diff
@@ -74,7 +72,7 @@ contract GammaPoolPriceOracle is BasePriceOracle, SafeOwnableUpgradeable {
 
     uint160 sqrtPriceX96 = toUint160(
       // sqrt((p0 * (10**token0.decimals()) * (1 << 96)) / (p1 * (10**token1.decimals()))) << 48
```

Comment:
@vminkov do you think we need to scale here by the decimals, as the gelato g uni oracle does?